Problem5/src/problem5/Survey.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package problem5;
+
+import java.io.*;
+import javax.swing.*;
+
+/**
+ * Survey object, used to store the results of surveys
+ * 
+ * @author Evan
+ */
+public class Survey
```

Comment:
Your helper Survey class should be implemented using OOP best practices.  Member variables should be private with appropriate getters and setters.